### PR TITLE
BEM avoid chaining modifiers blog post

### PR DESCRIPTION
**Pull request by acacheung**

### CODE REVIEW

File: source/posts/2015-01-05-avoid-chaining-modifiers.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+tags: css
+social: true
+comments: true
+published: false
+---
+
+At DockYard, we use the BEM methodology for naming our CSS classes. If
+you aren&rsquo;t already familiar with BEM, it would be helpful to take a look at
+[Harry Robert&rsquo;s MindBEMding blog post](http://csswizardry.com/2013/01/mindbemding-getting-your-head-round-bem-syntax/)
+and the [BEM website](https://bem.info/) first.
```
Comment (michaeldupuisjr) [inline source/posts/2015-01-05-avoid-chaining-modifiers.md]: I still think you should give a 1 sentence, high-level overview of what BEM is.

File: source/posts/2015-01-05-avoid-chaining-modifiers.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+When I started looking into using BEM, I thought it would definitely work well
+for large web applications. The more rules we could put in place
+regarding naming conventions, the better. What I didn&rsquo;t understand
+was what to do when an element could be considered an element of a block *or* a
+modifier. If I had a button that was in the footer and it was styled differently
+than other buttons in the website, should it be named `.footer__button`
+or `.button--footer`? Arguments could be made for both sides, so is one better
+than the other?
```
Comment (michaeldupuisjr) [inline source/posts/2015-01-05-avoid-chaining-modifiers.md]: Excellent, easy to understand example!

**Comment by michaeldupuisjr** [comment]
I love the post because it's short, easy to understand for someone not familiar with BEM, and the examples are excellent! :panda_face:

File: source/posts/2015-01-05-avoid-chaining-modifiers.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+well? To avoid confusion, I would call it `.footer__button--social`
+and not chain my modifiers. This doesn&rsquo;t mean we
+shouldn&rsquo;t use multiple modifiers in a class name at all.
+`.person--female__hand--right` is still fair game. It&rsquo;s clear that female
+is modifying person and right is modifying hand.
+
+## TL;DR
+If I had an element that could potentially be either an element of a
+block or modifying an element, the latter is better because
+chaining modifiers can be confusing.
```
Comment (NetOperatorWibby) [inline source/posts/2015-01-05-avoid-chaining-modifiers.md]: I remember this from our convo a few weeks ago. Nice and succinct.

File: source/posts/2015-01-05-avoid-chaining-modifiers.md (hunk 1 above)
Comment (acacheung) [inline source/posts/2015-01-05-avoid-chaining-modifiers.md]: kk will do

File: source/posts/2015-01-05-avoid-chaining-modifiers.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+twitter: 'acacheung'
+github: 'acacheung'
+author: 'Amanda Cheung'
+tags: css
+social: true
+comments: true
+published: false
+---
+
+At DockYard, we use the BEM methodology for naming our CSS classes. If
```
Comment (bcardarella) [inline source/posts/2015-01-05-avoid-chaining-modifiers.md]: You should link out to something that explains BEM

Comment (bcardarella) [inline source/posts/2015-01-05-avoid-chaining-modifiers.md]: Nevermind, didn't read the next line :p

**Comment by bcardarella** [comment]
:+1:

**Pull request merged**